web/vite.config.ts

```diff
@@ -1,38 +1,30 @@
 import { fileURLToPath, URL } from "node:url";
-
 import react from "@vitejs/plugin-react-swc";
 import { VitePWA } from "vite-plugin-pwa";
-import { defineConfig, loadEnv } from "vite";
+import { ConfigEnv, defineConfig, loadEnv, splitVendorChunkPlugin } from "vite";
+
 // https://vitejs.dev/config/
-export default ({ mode }: { mode: any }) => {
+export default ({ mode }: ConfigEnv) => {
   // early load .env file
-  process.env = { ...process.env, ...loadEnv(mode, process.cwd()) };
   // import.meta.env.VITE_NAME available here with: process.env.VITE_NAME
+  process.env = { ...process.env, ...loadEnv(mode, process.cwd()) };
 
   return defineConfig({
     base: "",
-    plugins: [react(), VitePWA({
+    plugins: [react(), splitVendorChunkPlugin(), VitePWA({
       registerType: "autoUpdate",
       injectRegister: "inline",
       scope: "{{.BaseUrl}}",
       // strategies: "injectManifest",
       useCredentials: true,
-      includeAssets: [
-        "favicon.svg",
-        "favicon.ico",
-        "robots.txt",
-        "logo.png",
-        "apple-touch-icon-*.png",
-        "manifest.webmanifest",
-        "assets/**/*"
-      ],
```

Comment:
I see the docs, the `includeAssets` by default already gets public folder and icons... I tested with and without this code and the generated manifest had no changes.

Reply:
Actually let me do more tests